planutils/packages/TEMPLATE/run

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+# whatever command-line method needs to be used to run this package
```

Comment:
There are a couple of files like this that do not have newlines in the last line. You can probably set up your editor to add those automatically (and remove trailing spaces at the same time).